explorer/ast/static_scope.h

```diff
@@ -35,11 +36,18 @@ class StaticScope {
   };
 
   // Construct a root scope.
-  StaticScope() = default;
+  explicit StaticScope(Nonnull<TraceStream*> trace_stream)
+      : ast_node_(std::nullopt), trace_stream_(trace_stream){};
```

Comment:
```suggestion
      : ast_node_(std::nullopt), trace_stream_(trace_stream) {}
```
I think the trailing semicolon is confusing clang-format here.